### ISSUE TITLE
Social media icons do not announce that they open in a new window

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#202]
This can be confusing for screen reader users. Icons don't have predictable behavior (opening in the same tab) and users cannot use back command as expected. Add aria-label=Opens in a new tab or aria-labeledby, or add text to the link "opens in a new tab"

Related to the issue #53
